.travis.yml

```diff
@@ -1,4 +1,4 @@
+sudo: false
 rvm:
-  - ree
-  - 1.9.2
-  - 1.9.3
```

Comment:
2.0.0 is nit dead yet, so I'd try to support it if it does not break a bunch ...